### ISSUE TITLE
Enhancement suggestion: Post/Page view counter

### ISSUE THREAD

**Issue by JayKayCZ** [deconf/Google-Analytics-Dashboard-for-WP#31]
Hello,
I have suggestion for new functionality. There is plenty wordpress post views counters, but none connected directly to google analytics. What about to implement options to integrate/hide some counters directly to posts page or pages page in wp administration? Counters may be the most common top choose, like "total views" or "total unique views".
Thank you for this amazing plugin!
Regards,
Jiri

**Comment by elvismdev** [comment]
Agree, it will be good to have a set of shortcodes like described in https://github.com/deconf/Google-Analytics-Dashboard-for-WP/issues/32

e.g. a shortcode to display the realtime stats/visitors in the frontend. Having this data managed by Google Analytics and make it available to display via a shortcode for the frontend would be a nice feature. isn't there a workaround for this actually?
